app/src/main/java/com/keylesspalace/tusky/db/Converters.kt

```diff
@@ -61,13 +63,13 @@ class Converters {
         return str?.split(";")
                 ?.map {
                     val data = it.split(":")
-                    createTabDataFromId(data[0], data.drop(1))
+                    createTabDataFromId(data[0], data.drop(1).map { s -> URLDecoder.decode(s, "UTF-8") })
```

Comment:
Can you try moving the url decoding somewhere else, so it doesn't run for tabs that don't need it? E.g. to `TabData.createTabDataFromId`